exporter/signalfx/signalfx_test.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2017, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package signalfx
+
+import (
+	"testing"
+)
+
+func TestExporterTokenSet(t *testing.T) {
+	_, err := NewExporter(Options{Token: "token"})
+	if err != nil {
+		t.Fatalf("shold not have generater error with token not null. err: %T", err)
+	}
+}
+
+func TestExporterTokenNotSet(t *testing.T) {
+	_, err := NewExporter(Options{})
+	if err == nil {
+		t.Fatal("shold have generater error with token null")
+	} else {
+		if err.Error() != "token cannot be empty on options signalfx.Options" {
+			t.Fatal("expected empty token error message")
+		}
+	}
+}
```

Comment:
I think test coverage could be improved here

Reply:
I'm currently testing all Exported methods from the signalfx package. More tests would actually involve testing the request from the signalfx client and I believe that this is not what is intended.

Reply:
You're not testing the most important thing here: the conversion of OpenCensus views into metrics sent to SignalFx, to confirm that you're sending in the expected metric names, dimensions and values. For reference: https://github.com/census-instrumentation/opencensus-java/blob/master/exporters/stats/signalfx/src/test/java/io/opencensus/exporter/stats/signalfx/SignalFxSessionAdaptorTest.java

Reply:
Yes, I added these tests.

Reply:
Check the newest version please.